parlai/core/dict.py

```diff
@@ -71,8 +71,9 @@ class DictionaryAgent(Agent):
     default_maxngram = -1
     default_minfreq = 0
     default_null = '__NULL__'
-    default_eos = '__EOS__'
     default_unk = '__UNK__'
+    default_eos = '__EOS__'
+    default_go = '__GO__'
```

Comment:
So SOS instead of GO throughout? SOS just looks kind of odd but I'm happy to go with it